scripts/live-connector-e2e/run-windows.ps1

```
@@ -0,0 +1,344 @@
+    $lines = @(Get-EventLines $Path)
+    if ($Since -ge $lines.Count) { return $false }
+    foreach ($line in $lines[$Since..($lines.Count - 1)]) {
+        try {
+            $eventRecord = $line | ConvertFrom-Json
+            if ($eventRecord.event_type -in @('tool_invocation', 'llm_prompt', 'llm_response') -and $line.ToLowerInvariant().Contains($Name.ToLowerInvariant())) { return $true }
+        } catch { continue }
+    }
+    return $false
+}
+
+function Write-Result([string]$Event, [string]$Status, [string]$Detail = '') {
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'Event' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)